container/container_windows.go

```diff
@@ -10,6 +10,10 @@ import (
 	containertypes "github.com/docker/docker/api/types/container"
 )
 
+const (
+	containerSecretMountPath = `C:\ProgramData\Docker\secrets`
```

Comment:
Currently, yes. We kept the option open to allow arbitrary paths in future (i.e., to allow the user to specify where the secrets should be available inside the containers)